internal/cmd/server/create.go

```diff
@@ -261,6 +261,11 @@ func createOptsFromFlags(
 	primaryIPv6IDorName, _ := flags.GetString("primary-ipv6")
 	protection, _ := flags.GetStringSlice("enable-protection")
 
+	// check if ssh-key flag was set, otherwise set to defaults
+	if ctx := s.Config().ActiveContext; ctx != nil && !flags.Changed("ssh-key") {
+		sshKeys = ctx.SSHKeys
+	}
```

Comment:
I think we should add this to the help output for this command. Same for `enable-rescue`.

What do you think about logging an info text that the default keys were used to create the token? Otherwise the customer might have forgotten about it and expects an email with the password.

Reply:
Yes, that sounds useful. I'll add that.